lib/Common/DataStructures/FixedBitVector.h

```diff
@@ -681,6 +681,19 @@ class BVStatic
         return true;
     }
 
+    bool IsAllSet() const
+    {
+        for (BVIndex i = 0; i < wordCount; i++)
+        {
+            if (!this->data[i].IsFull())
```

Comment:
>IsFull() [](start = 31, length = 8)

is this correct? what if there's unused bits at the end? IsAllClear is OK because I think we zeroed while initializing.

I think we need to check with rangeSize for last word, in case rangeSize%wordSize !=0